mlir/lib/Quantum/Transforms/QubitUnitaryFusion.cpp

```diff
@@ -0,0 +1,110 @@
+// Copyright 2023 Xanadu Quantum Technologies Inc.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#define DEBUG_TYPE "qubit-unitary-fusion"
+
+#include "mlir/Dialect/Linalg/Ops.h"
```

Comment:
I think this file does not exist. The closest I can find is `#include "mlir/Dialect/Linalg/IR/Linalg.h"`, so I would go find in the mlir repo where the operations you want are.